core/ui.libs/src/style/text.scss

```diff
@@ -24,6 +24,13 @@
     color: cssVar('text-color');
   }
 
+  h1, h2, h3, h4, h5 {
+    font-weight: bold !important;
+    i.mdi {
+      font-size: cssVar('h1-font-size');
```

Comment:
there should be no need for this, it should rather be "inherit" so it always adapts to the font size.